spec/models/user_spec.rb

```diff
@@ -4,7 +4,8 @@
 
 RSpec.describe User, type: :model do
   it 'is valid with valid attributes' do
-    user = User.new(name: 'Olivia', email: 'olivia@aivilo.com')
+    user = User.new(name: 'Olivia', email: 'olivia@aivilo.com',
+                    password: 'testing')
```

Comment:
For these spec, I find [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers). They let you write the spec as 
```
  it { is_expected.to validate_presence_of(:name) }
```